src/integrationTest/resources/features/post_logon_audit.feature

```diff
@@ -14,3 +14,14 @@ Feature: The application's POST logon audit endpoint
     Given LAU IdAm backend application is healthy
     When I POST "/audit/logon" endpoint with invalid body parameter using s2s
     Then http bad request response is returned for POST logon
+
+  Scenario: The backend is able to process 10 different idam logons POST requests in 10 different threads
+    Given LAU IdAm backend application is healthy
+    When I POST 10 request to "/audit/logon" endpoint with s2s in asynchronous mode
+    Then  logon response body is returned for all ten requests
+
+  Scenario: The backend is able to process 10 different idam logons POST requests with some failures
```

Comment:
I would be also good idea to update one of the existing integrations tests to prove that feign clients are called once on failures

Reply:
I'm not sure you understood my comment, please either update one of the existing GET integration tests to prove that GET endpoints are called once or create a new one.